### DISCUSSION TITLE
<title>Verify your account

### DISCUSSION THREAD

**Discussion by Animeman1999** [community/community#111357] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I am attempting to create an account for my school email. I am told an account all ready exists. I attempt to log in, don't know the credential. Sent to verify account which is very annoying having to solve 8 puzzles. This is really stupid. Any ways after wasting my time solving puzzles having solve this 8 puzzle sequence 4 times I wanted to give you feedback on this awful puzzle system. As someone who has  dyslexia, I feel like suing you for not being ADA compliant by forcing users to use this puzzle systems to get you to send a password reset. After half an hour of wasting my time I get a message unable to verify your captcha response. Thus I am unable to reset my password.

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket - New Ticket

Good Luck!

**Comment by ettaboyle** [comment]
Hi @Animeman1999, Echoing what @gil-son posted: The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

We cannot help or respond to any account specific questions on this public forum.
